lib/views/customer_details.erb

```diff
@@ -158,6 +169,19 @@
               <%end%>
             </div>
             <h3 class="pt-1 pb-1"> Billing Details </h3>
+            <div class="form-check p-2 pl-4 pb-3">
+                <input 
+                  class="form-check-input same-address" 
+                  type="checkbox" 
+                  value="true" 
+                  id="same-address" 
+                  name="same_address"
+                  />
+              <label class="form-check-label same-address" for="same-address">
+                Billing details are the same as shipping details.
+              </label>
+            </div>
+              <div class="to-hide">
```

Comment:
Indentation seems a bit hit and miss through this file. Might be worth running it through a html formatter?